AnnService/src/IndexSearcher/main.cpp

```diff
@@ -69,6 +70,8 @@ class SearcherOptions : public Helper::ReaderOptions
     int m_debugQuery = -1;
 
     bool m_enableADC = false;
+
+    int m_outputformat = 0;
```

Comment:
Can we use SPTAG::VectorFileType instead

Reply:
It is the result output type not the input vector type